cves/kernel/CVE-2017-5576.yml

```diff
@@ -387,37 +394,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
```

Comment:
Wouldn't this vulnerability apply to distrust_input? This looks like it takes parameters from the user